Public/Src/FrontEnd/Script/Ambients/Transformers/AmbientTransformer.Process.cs

```diff
@@ -545,7 +547,10 @@ private void ProcessExecuteArguments(Context context, ObjectLiteral obj, Process
             var errorRegex = Converter.ExtractString(obj, m_executeErrorRegex, allowUndefined: true);
             if (errorRegex != null)
             {
-                processBuilder.ErrorRegex = new RegexDescriptor(StringId.Create(context.StringTable, errorRegex), RegexOptions.None);
+                var enableMultiLine = Converter.ExtractOptionalBoolean(obj, m_executeErrorRegexEnableMultiLineMatches);
+                processBuilder.ErrorRegex = new RegexDescriptor(
+                    StringId.Create(context.StringTable, errorRegex),
+                    enableMultiLine == true ? RegexOptions.Singleline : RegexOptions.None);

```

Comment:
>RegexOptions.Singleline [](start = 46, length = 23)

Should we just instruct users to specify the inline processing options instead of setting the enum in the engine itself?